src/core/manager.c

```diff
@@ -1717,13 +1717,26 @@ static void manager_ready(Manager *m) {
         /* Let's finally catch up with any changes that took place while we were reloading/reexecing */
         manager_catchup(m);
 
+        /* Create a file which will indicate when the manager was reloaded for the last time. */
+        if (m->n_reloading == 0) {
+                m->reloaded_realtime_timestamp = m->reloading_start_monotonic_timestamp == 0
```

Comment:
why store reloaded_realtime_timestamp in the Manager object? you just need one I figure.

also, why the weird conversion? if you need the realtime timestamp, then pick up the realtime timestamp

Reply:
At first we wanted to cover times when the system time is changed, but there are still cases when this could fail. It would be better to use a simple realtime timestamp.